### PR TITLE
plugins:foreman_docker - Release 1.4.1

### PR DESCRIPTION
**Pull request by eLobato**

### CODE REVIEW

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 Summary:    A Foreman plugin for Docker container management
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    1.4.0
+Version:    1.4.1
 Release:    2%{?foremandist}%{?dist}
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: reset this

File: rubygem-foreman_docker/rubygem-foreman_docker.spec [hunk 2]
```diff
@@ -118,6 +118,9 @@ cp -a .%{gem_dir}/* \
 exit 0
 
 %changelog
+* Tue Aug 18 2015 Daniel Lobato Garcia <elobatocs@gmail.com> 1.4.1-1
+- plugins:foreman_docker - Release 1.4.1 (elobatocs@gmail.com)
```
Comment (domcleal) [inline rubygem-foreman_docker/rubygem-foreman_docker.spec]: Still preferred if you can leave this out, since tito generates it.

**Comment by domcleal** [comment]
The gem hasn't been published, please push it.

**Comment by domcleal** [comment]
Thanks @eLobato, building in nightlies.  Any other release?

**Comment by eLobato** [comment]
@domcleal Yes, 1.7, 1.8, 1.9 too please

**Comment by domcleal** [comment]
Building for both RPM and deb for all three releases.

**Pull request merged**